docs/articles/sde/introduction_to_sde.rst

```diff
@@ -55,6 +56,13 @@ the code which will make them pass. It's really important to have automatic
 tests which will test for any regressions if someone changes parts of the
 code.
 
+Tools list
+^^^^^^^^^^
+Following is a list of tools already installed into the SDE.
+
+* QT_Creator: IDE with a template wizard provided to speed the process. 
+* DLT_Viewer: Visual application to interact with the dlt-daemon.
```

Comment:
I believe the tools are called Qt Creator and DLT Viewer.

Reply:
You right, QtCreator is also listed on the spelling_wordlist.